### ISSUE TITLE
feat: add section for the installation and usage

### ISSUE THREAD

**Issue by roberthgnz** [roberthgnz/buzz-notify-docs#3]

**Comment by hardy07** [comment]
can give me a brief about the issue?

**Comment by roberthgnz** [comment]
> can you give me a brief about the issue?

Of course... I want to add these sections based on the current readme instructions:

https://github.com/roberthgnz/buzz-notify#readme
